backend/app.js

```diff
@@ -0,0 +1,16 @@
+
+
+const express = require('express');
+
+
+const app = express();
+
+const trainerRoutes = require('./routes/trainer.routes');
+
+app.use('/api/admin', trainerRoutes);
+
+
+
+app.listen(6000, () => {

```

Comment:
PORT number should be a constant and should be stored in an env. variable.
Please refer packages like `dotenv`

Reply:
+ Log messages should also print port number.
e.g : `Server is listening on PORT : ${PORT}`